src/OpenTelemetry.Exporter.Geneva/README.md

```diff
@@ -157,6 +157,57 @@ values.
   * `ILogger<MyPartner.Product.Thing>`: This is marked as pass-through ("*") so
     it will be sanitized as "MyPartnerProductThing" table name
 
+#### How to configure GenevaExporterOptions using dependency injection
+
+##### Tracing
+
+> [!NOTE]
+> In this example named options ('GenevaTracing') are used. This is because
+> `GenevaExporterOptions` is shared by both logging & tracing. In a future
+> version named options will also be supported in logging so it is recommended
+> to use named options now for tracing in order to future-proof this code.
```

Comment:
```suggestion
> In this example named options ('GenevaTracing') are used. This is because
  `GenevaExporterOptions` is shared by both logging & tracing. In a future
  version named options will also be supported in logging so it is recommended
  to use named options now for tracing in order to future-proof this code.
```